training/training/routes/image/image.py

```diff
@@ -0,0 +1,54 @@
+from typing import Literal, Optional
+from django.http import HttpRequest
+from ninja import Router, Schema
+from training.core.criterion import getCriterionHandler
+from training.core.dl_model import DLModel
+from training.core.datasets import ImageDefaultDatasetCreator
+from torch.utils.data import DataLoader
+from training.core.optimizer import getOptimizer
+from training.core.trainer import ClassificationTrainer
+from training.routes.image.schemas import ImageParams
+from training.core.authenticator import FirebaseAuth
+
+router = Router()
+
+
+@router.post("", auth=FirebaseAuth())
+def imageTrain(request: HttpRequest, imageParams: ImageParams):
+    if imageParams.default:
+        dataCreator = ImageDefaultDatasetCreator.fromDefault(
+            imageParams.default, imageParams.test_size, imageParams.shuffle
+        )
+        train_loader = DataLoader(
+            dataCreator.createTrainDataset(),
+            batch_size=imageParams.batch_size,
+            shuffle=False,
```

Comment:
is shuffle an arg the user can specify @farisdurrani ?

Reply:
@codingwithsurya looks like shuffle is an arg the user can specify.

Reply:
sounds good. will incorporate. TO-DO.